gym/vector/async_vector_env.py

```diff
@@ -53,7 +53,7 @@ class AsyncVectorEnv(VectorEnv):
         Only available in Python 3.
     """
     def __init__(self, env_fns, observation_space=None, action_space=None,
-                 shared_memory=True, copy=True, context=None):
+                 shared_memory=True, copy=True, context=None, daemon=True, worker=None):
```

Comment:
When do you want to override the worker?

Reply:
currently, worker calls reset every time environment returns done, I want to override worker to replace that logic with our AsyncResetOnDone logic

Reply:
The irony is that there was a PR with a very similar change in baselines, and I closed it because I was not motivated enough by the example :)